dspace-rest/src/main/java/org/dspace/rest/BitstreamResource.java

```diff
@@ -165,7 +165,6 @@ public Bitstream getBitstream(@PathParam("bitstream_id") String bitstreamId, @Qu
         {
             context = createContext(getUser(headers));
             org.dspace.content.Bitstream dspaceBitstream = findBitstream(context, bitstreamId, org.dspace.core.Constants.READ);
-            List<org.dspace.authorize.ResourcePolicy> resourcePolicies = authorizeService.getPolicies(context, dspaceBitstream);
             //TODO why isn't the above used...
```

Comment:
Perhaps the comment should be removed as well, since it appears to have referred to the removed assignment and would now be confusing.